### PR TITLE
[usecase] adds example of copying a CSV to Snowflake

### PR DESCRIPTION
**Pull request by cmpadden**
## Summary & Motivation

- No external dependencies (e.g. pandas)
- Uses `SnowflakeResource`
- Uses `COPY INTO` statement

## How I Tested These Changes

- Use case preview

### CODE REVIEW

File: examples/use_case_repository/use_case_repository/guides/copy_csv_to_snowflake.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+   ```sh
+   dagster dev
+   ```
+
+2. Open the Dagster UI at [http://127.0.0.1:3000](http://127.0.0.1:3000).
+
+3. Find the `load_csv_to_snowflake` asset and click on it.
+
+4. Click on the "Launchpad" tab.
```
Comment (PedramNavid) [inline examples/use_case_repository/use_case_repository/guides/copy_csv_to_snowflake.md:108]: what is this 'launchpad'

**Review by PedramNavid — CHANGES_REQUESTED** [review]
Small change, otherwise good to go. 
I assume you tested the code and it works? it looks good to me.

File: examples/use_case_repository/use_case_repository/guides/copy_csv_to_snowflake.md (hunk 1 above)
Comment (cmpadden) [reply examples/use_case_repository/use_case_repository/guides/copy_csv_to_snowflake.md]: Simplified the instructions

**Comment by cmpadden** [comment]
> Small change, otherwise good to go. I assume you tested the code and it works? it looks good to me.

Confirmed working as expected!

- Updated the example to better adhere to Snowflake's instructions.
https://docs.snowflake.com/en/sql-reference/sql/copy-into-table

- Confirmed that data loaded as expected in our test environment:
image

**Review by PedramNavid — APPROVED** [review]
looks swell

**Pull request merged**